utils/_context/containers.py

```diff
@@ -162,6 +198,9 @@ def start(self) -> Container:
         self.wait_for_health()
         self.warmup()
 
+    def depend_on(self, container):
```

Comment:
`TestedContainer.depends_on.append()` is fine, and I don't think you use `TestedContainer.depend_on` anywhere. =>  you can remove it.